pallets/orders/src/lib.rs

```diff
@@ -141,11 +162,13 @@ pub mod pallet {
 			let who = Self::ensure_signed_or_para(origin)?;
 
 			// TODO: charge order creation cost
+			// Where shall we send the order creation cost to? To Treasury?
```

Comment:
Yeah, this shouldn't be defined in the pallet. This should be configurable through the pallet `Config`.